src/mocks/jest-server.js

```diff
@@ -0,0 +1,5 @@
+import { setupServer } from 'msw/node'; // eslint-disable-line import/extensions
```

Comment:
Why can't extension?

Reply:
I'm not entirely sure why this is the case, but here's the error stack I get:
```
    Cannot find module 'msw/node.js' from 'src/mocks/jest-server.js'

    Require stack:
      src/mocks/jest-server.js
      jest.config.setup.files.after.env.js

    > 1 | import { setupServer } from 'msw/node.js';
```

Reply:
This is, I think, complaining because it _looks_ like a relative import from our own code because it has a `/` in the path. The linter doesn't complain about other module imports without extensions.